src/components/SearchAndFilter/FilterPanelSection/FilterPanelSection.test.js

```diff
@@ -10,12 +10,16 @@ const sampleData = {
 
 describe("Filter panel section", () => {
   it("renders", () => {
-    const wrapper = shallow(<FilterPanelSection data={sampleData} />);
+    const wrapper = shallow(
+      <FilterPanelSection data={sampleData} searchTerm="" />
```

Comment:
Why is `searchTerm` a required prop here? Can it not be set to a default value of an empty string?

Reply:
Updated :)